cfme/intelligence/reports/ui_elements.py

```diff
@@ -1,802 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
are you sure you really need to remove this file in first PR ? 
As far as I understand all tests which use this file will be broken until the rest of PRs is merged.